### PR TITLE
feat(codeeditor): support shell language

### PR DESCRIPTION
**Pull request by andrewsemchism**
CodeSandbox logoCodeSandbox logo&nbsp; Open in CodeSandbox <a href="https://codesandbox.io/p/github/andrewsemchism/sandpack/language-shell?workspace=%7B%22gitSidebarPanel%22:%22PR%22,%22sidebarPanel%22:%22GIT%22%7D">Web Editor</a> | <a href="https://codesandbox.io/p/vscode?owner=andrewsemchism&repo=sandpack&branch=language-shell">VS Code</a>

## What kind of change does this pull request introduce?

Add support for shell syntax highlighting.

## What is the current behavior?

Shell syntax highlighting is not supported.

## What is the new behavior?

Files with the extensions `.sh`, `.shell`, `.bash`, and `.zsh` will have shell syntax highlighting applied to them.

## What steps did you take to test this? This is required before we can merge, make sure to test the flow you've updated.

1. Added shell example to storybook.

## Checklist

- [ ] Documentation; N/A
- [X] Storybook (if applicable);
- [ ] Tests;
- [X] Ready to be merged;

### CODE REVIEW

**Comment by andrewsemchism** [comment]
@danilowoz Could you re-run the failed Integration tests? I'm not sure what caused that failure. Also, would it possible to get some guidance on fixing the Chromatic deployment "missing project token" failure? Thank you.

**Comment by jodyheavener** [comment]
Hey @danilowoz! I just wanted to check in here to see if anything else is needed for a review on this PR. I'd love to start using this in the our usage of Sandpack 🙂

File: sandpack-react/src/components/CodeEditor/utils.ts [hunk 1]
```diff
@@ -189,6 +192,11 @@ export const getLanguageFromFile = (
       return "css";
     case "md":
       return "markdown";
+    case "sh":
+    case "shell":
+    case "bash":
+    case "zsh":
```
Comment (mrjones2014) [inline sandpack-react/src/components/CodeEditor/utils.ts:198]:
```suggestion
    case "zsh":
    case "fish":
```

**Comment by danilowoz** [comment]
Implemented additional language support on https://github.com/codesandbox/sandpack/pull/570

**Pull request closed without merging**